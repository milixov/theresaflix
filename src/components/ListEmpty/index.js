import React from 'react';
import {StyleSheet, Text} from 'react-native';

const ListEmpty = ({text}) => {
    return <Text style={styles.container}>{text}</Text>;
};

const styles = StyleSheet.create({
    container: {
        flex: 1,
        marginHorizontal: 16,
        fontSize: 18,
        fontWeight: 'bold',
        color: 'gray',
    },
});

export default ListEmpty;
