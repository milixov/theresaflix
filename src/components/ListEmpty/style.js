import THEME from '../../style';
import {StyleSheet} from 'react-native';

export const styles = StyleSheet.create({
    container: {
        flex: 1,
        marginHorizontal: 16,
        fontSize: 18,
        fontWeight: 'bold',
        color: THEME.COLOR.GRAY,
    },
});
