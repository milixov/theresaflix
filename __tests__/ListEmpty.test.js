import React from 'react';
import ListEmpty from '../src/components/ListEmpty';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<ListEmpty />).toJSON();
    expect(tree).toMatchSnapshot();
});
