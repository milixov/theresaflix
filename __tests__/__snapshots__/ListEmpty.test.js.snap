// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "gray",
      "flex": 1,
      "fontSize": 18,
      "fontWeight": "bold",
      "marginHorizontal": 16,
    }
  }
/>
`;
